povm_toolbox/post_processor/ppp_empirical_frequencies.py

```diff
@@ -0,0 +1,103 @@
+# (C) Copyright IBM 2024.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""TODO."""
+
+from __future__ import annotations
+
+import numpy as np
+from qiskit.quantum_info import DensityMatrix, SparsePauliOp, Statevector
+
+from povm_toolbox.post_processor import POVMPostProcessor
+from povm_toolbox.quantum_info import ProductPOVM
+from povm_toolbox.quantum_info.product_dual import ProductDUAL
+
+
+class PPPEmpiricalFrequencies(POVMPostProcessor):
```

Comment:
Does `PPP` stand for **P**OVM**P**ost**P**rocessor?

Not sure whether we need that in all these names... We don't do that with `ClassicalShadows` either (for example).
Maybe `EmpiricalFrequencies` is enough, given that this can only be imported from the `post_processor` module to begin with?